src/filesystem/impls/lib/log.js

```diff
@@ -0,0 +1,13 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define */
+define(function (require, exports, module) {
+    "use strict";
+
+    exports.error = function(msg) {
+        console.error('[Brackets BlobUtils] Error: ' + msg);
```

Comment:
Because you use this logging outside the `BlobUtils` file, I wonder if you should change this to `Filer Filesystem` or something?